.github/workflows/nightly-TICS.yaml

```
@@ -0,0 +1,36 @@
+name: Nightly TICS scan
+
+on:
+  schedule:
+    - cron: '44 1 * * *'
```

[nit]: Can you comment the time of day that this action will run? Helpful for anyone coming into the project that might not be super familiar with cron syntax:

```suggestion
    - cron: '44 1 * * *' # Every day at 1:44 AM
```

Also, is there a specific reason why you chose for this action to run at 1:44 AM? Is it because everyone else chooses to have their cron job run at 12:00 AM on GitHub? :sweat_smile:

---

Yeah it's just a random time that was taken from https://github.com/medvied/charm-ceph-mon/blob/master/.github/workflows/nightly-TICS.yaml.
It's not recommended to schedule jobs at "standard" times like 00:00 am or 12:00 pm:
> https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule
The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. If the load is sufficiently high enough, some queued jobs may be dropped. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.